devel/ci/Dockerfile-rawhide

```diff
@@ -8,48 +8,60 @@ RUN dnf install --disablerepo rawhide-modular -y \
     findutils \
     git \
     liberation-mono-fonts \
+    python2-yaml \
+    python3-dnf \
+    python3-yaml \
     make \
     python2-click \
     python2-fedora \
+    python2-kitchen \
```

Comment:
Kitchen is not used by the client, so we shouldn't need it here.

Reply:
I based my filtering on the specfile's dependencies, so it needs to be removed there too probably.